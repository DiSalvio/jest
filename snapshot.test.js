import User from './user.js'

const user = new User({
  firstName: 'Joey',
  lastName: 'Spumoni',
  age: 42,
  job: null
})

it('matches user snapshot', () => {
  expect(user).toMatchSnapshot()
})
