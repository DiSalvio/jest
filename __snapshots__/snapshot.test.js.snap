// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches user snapshot 1`] = `
User {
  "age": 42,
  "firstName": "Joey",
  "job": null,
  "lastName": "Spumoni",
}
`;
